tests/behat/features/capabilities/event/filter-community-events.feature

```diff
@@ -0,0 +1,50 @@
+@api @javascript @event @stability @stability-1 @filter-community-events
```

Comment:
```suggestion
@api @javascript
```

These are the only tags we actually need :) The others don't do anything

Reply:
Fixed